recipes/hive.rb

```diff
@@ -24,7 +24,13 @@
 end
 
 hive_conf_dir = "/etc/hive/#{node['hive']['conf_dir']}"
-hive_data_dir = '/usr/lib/hive/lib'
+
+if node['hadoop']['distribution'] == 'hdp' && node['hadoop']['distribution_version'].to_f == 2.2
+  hive_data_dir = '/usr/hdp/current/hive-server2/lib'
+else
+  hive_data_dir = '/usr/lib/hive/lib'
+end
```

Comment:
Same comment about syntax. Also, this would require the hive-server2 package to be installed, wouldn't it? At this point, we only have the `hive` package and its directories. Correct?